source/jsonnet/england-wales/individual/blocks/employment/hours_worked.jsonnet

```diff
@@ -1,8 +1,23 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title) = {
-  title: title,
+local questionTitle(isProxy) = (
+  if isProxy then {
+    text: 'In <em>{person_name_possessive}</em> main job, how many hours a week do they usually work?',
+    placeholders: [
+      placeholders.personNamePossessive,
+    ],
+  }
+  else 'In your main job, how many hours a week do you usually work?'
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then 'If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>'
+  else 'If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>'
```

Comment:
I think there should be a full stop at the end of these sentences, but it's not in the spreadsheet - the spreadsheet is inconsistent though e.g. the travel to work content does end in a full stop.

Reply:
I've added full stops to all the ones you mentioned, I'll ask Donna to take a look today before I merge